pkg/apply/prune/prune_test.go

```diff
@@ -973,7 +888,7 @@ func TestPrune_PropagationPolicy(t *testing.T) {
 			eventChannel := make(chan event.Event, 1)
 			resourceCache := cache.NewResourceCacheMap()
 			taskContext := taskrunner.NewTaskContext(eventChannel, resourceCache)
-			err := po.Prune([]*unstructured.Unstructured{pdb}, []filter.ValidationFilter{}, taskContext, "test-0", Options{
+			err := po.Prune(object.UnstructuredSet{pdb}, []filter.ValidationFilter{}, taskContext, "test-0", Options{
```

Comment:
This minor cleanup could be in a separate PR, maybe with other cleanups in this test file.